history-of-quakers/modernized/17-chapter-15.adoc

```diff
@@ -409,57 +419,82 @@ the Lord has redeemed me and many more out of such things,
 and seeing I am engaged to appear at the next court session,
 I desire no further thing may be required of me.
 
-Judge: You must enter into bond this dangerous time, and therefore consider of it,
+[.discourse-part]
+_Judge:_ You must enter into bond this dangerous time, and therefore consider of it,
 and tell me now, or before the court session ends.
 
+[.small-break]
```

Comment:
Wasn't sure if this should be a [.small-break] or [.offset]...it looks a little wonky in the PDF either way.